cmake/binary_to_char_array.cmake

```diff
@@ -0,0 +1,51 @@
+# Copyright libCellML Contributors
```

Comment:
Might want to rename the file so that it doesn't use underscores (and thus be consistent with our other file names).

Reply:
493cee0